<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanDialog</class>
 <widget class="QDialog" name="PlanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>14</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="titleEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>你这段时间想做点什么</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="levelComb">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>时间区间</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QDateEdit" name="startDate">
            <property name="displayFormat">
             <string>yyyy/MM/dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>~</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="endDate">
            <property name="displayFormat">
             <string>yyyy/MM/dd </string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="contentText">
       <property name="placeholderText">
        <string>快记录下具体的内容-&gt;_-&gt;,等下就要忘记了偶^_^</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>事件类型</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="FCircle" name="dailyCircle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="flag">
             <string>日常生活</string>
            </property>
            <property name="radius">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FCircle" name="workCircle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="color">
             <string>#1b7cff</string>
            </property>
            <property name="flag">
             <string>工作</string>
            </property>
            <property name="radius">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FCircle" name="studyCircle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="color">
             <string>#d90d0d</string>
            </property>
            <property name="flag">
             <string>学习</string>
            </property>
            <property name="radius">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FCircle" name="playCircle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="color">
             <string>#f79f1f</string>
            </property>
            <property name="flag">
             <string>娱乐</string>
            </property>
            <property name="radius">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FCircle" name="travelCircle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="color">
             <string>#3ab155</string>
            </property>
            <property name="flag">
             <string>旅游</string>
            </property>
            <property name="radius">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FButton" name="showBtn">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>8</height>
             </size>
            </property>
            <property name="borderRadius">
             <number>4</number>
            </property>
            <property name="bgColor">
             <string>#fff</string>
            </property>
            <property name="maxShadow">
             <number>15</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="FButton" name="okBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
         <property name="borderRadius">
          <number>2</number>
         </property>
         <property name="maxShadow">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FButton" name="cancelBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
         <property name="borderRadius">
          <number>2</number>
         </property>
         <property name="bgColor">
          <string>#404244</string>
         </property>
         <property name="maxShadow">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FButton</class>
   <extends>QPushButton</extends>
   <header>fbutton.h</header>
  </customwidget>
  <customwidget>
   <class>FCircle</class>
   <extends>QLabel</extends>
   <header>fcircle.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
